### PR TITLE
[DR-6] Footer with links

### PR DESCRIPTION
**Pull request by blurbyte**
[DR-6](https://digitalrig.atlassian.net/browse/DR-6) Footer with links

Screenshot:
Screenshot 2019-06-13 at 15 32 25

Implemented following things:
- [x] Wrapper for whole app
- [x] Adjust sizes of various elements
- [x] Footer with links (GitHub link shows Contribute label on desktop as well)

### CODE REVIEW

File: src/components/Header/Logo.js [hunk 1]
```diff
@@ -23,7 +24,7 @@ const Tagline = styled.p`
   margin: 0;
   padding: 0;
   color: ${colorBlack};
-  font-size: ${fontSmall};
+  font-size: 1.2rem;
```
Comment (mgolawski) [inline src/components/Header/Logo.js]: Why did you use straight values?

File: src/components/Header/Logo.js [hunk 2]
```diff
@@ -2,18 +2,19 @@ import React from 'react';
...
-import { colorBlack, spacingSmall, borderWidthThin, fontSmall } from '../../styles/designTokens';
+import { colorBlack, spacingSmall, borderWidthThin } from '../../styles/designTokens';
 import { Logo as LogoIcon } from '../Icons';
 
 const Link = styled(LinkBase)`
   display: flex;
   text-decoration: none;
   align-items: center;
   height: 100%;
+  cursor: pointer;
```
Comment (mgolawski) [inline src/components/Header/Logo.js]: Shouldn't it be pointer by default?

File: src/components/Footer/Layout.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+import { spacingMedium } from '../../styles/designTokens';
+
+const Layout = styled.footer`
```
Comment (mgolawski) [inline src/components/Footer/Layout.js]: I am more convinced by the `Wrapper` naming. We should pick single version.

File: src/components/Header/Logo.js (hunk 1 above)
Comment (blurbyte) [reply src/components/Header/Logo.js]: It is more part of logo than regular page layout. So it has custom value to just look good next to logo.

File: src/components/Header/Logo.js (hunk 2 above)
Comment (blurbyte) [reply src/components/Header/Logo.js]: For some reason without it cursor sometimes was changing to default one when hovering over tagline.

File: src/components/App/Wrapper.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+const Wrpper = styled.div`
```
Comment (mgolawski) [inline src/components/App/Wrapper.js]: Typo `Wrpper` -> `Wrapper`

**Pull request merged**